Cargo.toml

```diff
@@ -5,12 +5,14 @@ authors = ["Gedalia Kott <gedkott@gmail.com>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-[lib]
-name = "bencoding"
-path = "src/lib.rs"
 
 [[bin]]
 name = "bit_torrent"
 path = "src/main.rs"
 
 [dependencies]
+reqwest = { version = "0.10", features = ["blocking", "json"] }
```

Comment:
Use reqwest for http client. Nothing fancy. Didn't want to do go as deep as implementing HTTP. That's its own thing.